packages/webapp/app/app-boot.ts

```diff
@@ -0,0 +1,36 @@
+import type { Boot } from '@dailydotdev/shared/src/lib/boot';
```

Comment:
Slightly triggered by the file name `app-boot.ts` 🙈

Reply:
Renamed for the common wealth 😄